arch/common/CMakeLists.txt

```diff
@@ -2,6 +2,8 @@
 
 zephyr_library()
 
+zephyr_library_include_directories(include)
```

Comment:
@fkokosinski I've added this and the header path used in the source isn't relative anymore